README.md

```diff
@@ -56,18 +56,60 @@ See https://getcomposer.org/ for more information and documentation.
 
 ### Inlining Css
 
-This is how to use the `CssInliner` class:
+The most basic way to use the `CssInliner` class is to create an instance with
+the original HTML, inline the external CSS, and then get back the resulting
+HTML:
 
 ```php
-$visualHtml = \Pelago\Emogrifier\CssInliner::fromHtml($html)->inlineCss($css)->render();
+use Pelago\Emogrifier\CssInliner;
+
+…
+
+$visualHtml = CssInliner::fromHtml($html)->inlineCss($css)->render();
+```
+
+If there is no external CSS file and all CSS is located within a `<style>`
+node in the HTML, you can omit the `$css` parameter:
```

Comment:
I think this should be “element” not “node” and [MDN agrees](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style).  Also, as we support CSS in more than one `<style>` element, perhaps “element (or elements)”.

Reply:
Done.